tests/Invoke-DbaDiagnosticQuery.Tests.ps1

```diff
@@ -1,16 +1,93 @@
-$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+# test ouput directory to confirm creation of test files
+$commandname = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
 Write-Host -Object "Running $PSCommandpath" -ForegroundColor Cyan
-. "$PSScriptRoot\constants.ps1"
+. (Join-path (($PSScriptRoot, '.\' -ne '')[0]) "constants.ps1")  #if running in interactive console, cd to folder and you can run this without error
+write-host "loaded constants"
 
 Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
-    Context "Verifying output" {
+    BeforeAll {
+        $script:PesterOutputPath = "TestDrive:$commandName"
+
+
+        $database = "dbatoolsci_frk_$(Get-Random)"
+        $database2 = "dbatoolsci_frk_$(Get-Random)"
+        $server = Connect-DbaInstance -SqlInstance $script:instance2
+        $server.Query("CREATE DATABASE $database")
+        $server.Query("CREATE DATABASE $database2")
+
+    }
+    AfterAll {
+        $null = Get-DbaDatabase -SqlInstance $server -Databases @($database, $database2) | Remove-DbaDatabase -Confirm:$false
```

Comment:
Utilize similar pattern as the other test with `Query()` method for consistence and not having a dependency on other public functions.

Reply:
this is pretty much on every test....... let's not overcomplicate things, get-dbadatabase and remove-dbadatabase are not going to suddenly disappear